packages/rich-text/src/char-at.js

```diff
@@ -5,7 +5,7 @@
  * @param {Object} value Value to get the character from.
  * @param {string} index Index to use.
  *
- * @return {?string} A one character long string, or undefined.
+ * @return {string|undefined} A one character long string, or undefined.
```

Comment:
Is this a new recommendation? Do we have any docs on writing inline docs?

Reply:
As per the [`@type` tag documents](http://usejsdoc.org/tags-type.html) this is one of the issues I've identified:

- we should use `?string` if the type can be `null` or a `string`.
- we should use `string|undefined` if the type can be a `string` or `undefined`.

From the text description, I understood that we want the second.

Reply:
Nice, I like that it's clearer as well.